### ISSUE TITLE
v0.20.0 deps and belay select issue

### ISSUE THREAD

**Issue by raveslave** [BrianPugh/belay#133]
repro:

- did a clean venv + sourced it
- pip install belay
- belay [enter]
```
venv/lib/python3.11/site-packages/belay/cli/new.py", line 6, in <module>
    from packaging.utils import canonicalize_name
ModuleNotFoundError: No module named 'packaging'
```

then:   

- pip install packaging
- belay [enter] all good
- belay select
- note: this only happens when no board is found/connected
```
  File "venv/lib/python3.11/site-packages/belay/cli/select.py", line 119, in select
    device_index = select_table(
                   ^^^^^^^^^^^^^
  File "venv/lib/python3.11/site-packages/belay/cli/questionary_ext.py", line 104, in select_table
    raise ValueError("A list of choices needs to be provided.")
ValueError: A list of choices needs to be provided.
```

**Comment by BrianPugh** [comment]
Thanks for the report! For some reason I thought `packaging` was builtin; I'll open up a PR adding it as a dependency soon. I'll also be uncapping a lot of the dependency versions upper ends.

It appears that when no boards are found, the provided list is empty, and stuff errors out. I'll also add in logic to handle this more gracefully.

**Comment by BrianPugh** [comment]
Fixed in #135, will be a part of v0.20.1, which ill be releasing very soon

**Comment by BrianPugh** [comment]
sorry, I didn't notice your last comment, I'll also fix that right now; will be a part of v0.20.1 as well. That's a super simple bug that just slipped through the cracks (those have to be tested manually, not by CI).

**Comment by raveslave** [comment]
top!
